react/features/reactions/actions.any.ts

```diff
@@ -28,9 +26,8 @@ export function setReactionQueue(value: Array<ReactionEmojiProps>) {
  * Removes a reaction from the queue.
  *
  * @param {number} uid - Id of the reaction to be removed.
- * @returns {void}
  */
-export function removeReaction(uid: number) {
+export function removeReaction(uid: number): Function {
     return (dispatch: Function, getState: Function) => {
         const queue = getState()['features/reactions'].queue;
 
```

Comment:
Maybe add the filter return type below as well?

Reply:
Filter is different from map, so it's not possible (also, filter does not change the type of the inital array it just removes some elements)